ethcore/src/engines/clique/tests.rs

```diff
@@ -112,23 +112,23 @@ impl CliqueTester {
 	/// Get signers after a certain state
 	// This is generally used to fetch the state after a test has been executed and checked against
 	// the intial list of signers provided in the test
-	pub fn clique_signers(&self, hash: &H256) -> impl Iterator<Item = Address> {
+	pub fn clique_signers(&self, hash: &H256) -> impl Iterator<Item=Address> {
```

Comment:
More spaces always good.  I think this decreases code readability a bit.

Reply:
no worries, that is automatic formatting. 
I can change the setting if the spaces are expected in type bindings.
Unfortunately, I have not seen consistency in the codebase.
I would definitely prefer if parity were using cargo fmt and that's it :)

Reply:
just `cargo fmt` the entire codebase in one commit.  problem solved :)

Reply:
I will probably find a lot of new friends if I do that